presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1062,6 +1062,7 @@ public void testViewColumnAccessControl()
         // verify selecting from a view over a table requires the view owner to have special view creation privileges for the table
         assertAccessDenied(
                 "SELECT * FROM " + columnAccessViewName,
+                "View owner does not have sufficient privileges",
```

Comment:
can we match the full exception message here, not just a aubstring?
(the actual message has `: ...` after "privileges")

this would also make sure @electrum is happy about "Access Denied" being or not being repeated within the message

Reply:
actually I already had such version https://github.com/prestosql/presto/commit/d2e7d1a92d21d7927b7fdd4ef90c86aabab39ece
it requires adding `: Access Denied: ` somewhere within the error message which is not intuitive and error prone.
If you prefer this way I'll remove `Access Denied` text concatenation from internals of assertAccessDenied methods and add it to every text message during invocation of these methods. At least it will not be misleading.

Reply:
done